src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -187,6 +226,18 @@ public JedisCluster(Set<HostAndPort> clusterNodes, JedisClientConfig clientConfi
     super(clusterNodes, clientConfig, maxAttempts);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances. <br>
```

Comment:
Please don't use `Jedis-Cluster`. You can use `JedisCluster` or `Cluster` or `cluster`.